### PR TITLE
Pv2 2130 api configuration

### PR DESCRIPTION
**Pull request by ChughAkshay**

### CODE REVIEW

File: src/MatchedLearnerApi/Program.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
+using MatchedLearnerApi.Extensions;
 using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
```
Comment (hardik-desai-kcl) [inline src/MatchedLearnerApi/Program.cs:9]: why are we using nlog?

**Review by hardik-desai-kcl — CHANGES_REQUESTED** [review]

File: src/MatchedLearnerApi/Configuration/MatchedLearnerApiConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using MatchedLearnerApi.Interfaces;
+
+namespace MatchedLearnerApi.Configuration
+{
+    public class MatchedLearnerApiConfiguration : IMatchedLearnerApiConfiguration
+    {
+        public string DasPaymentsDatabaseConnectionString { get; set; }
+        public string LoggingConnectionString { get; set; }
```
[the remark this answers is not in the document]
Comment (ChughAkshay) [reply src/MatchedLearnerApi/Configuration/MatchedLearnerApiConfiguration.cs:12]: ✔

File: src/MatchedLearnerApi/Configuration/MatchedLearnerApiConfiguration.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        public string LoggingKey { get; set; }
```
[the remark this answers is not in the document]
Comment (ChughAkshay) [reply src/MatchedLearnerApi/Configuration/MatchedLearnerApiConfiguration.cs:13]: ✔

File: src/MatchedLearnerApi/Interfaces/IMatchedLearnerApiConfiguration.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MatchedLearnerApi.Interfaces
+{
+    public interface IMatchedLearnerApiConfiguration
+    {
+        string DasPaymentsDatabaseConnectionString { get; set; }
+        string LoggingConnectionString { get; set; }
```
[the remark this answers is not in the document]
Comment (ChughAkshay) [reply src/MatchedLearnerApi/Interfaces/IMatchedLearnerApiConfiguration.cs:11]: ✔

File: src/MatchedLearnerApi/Extensions/IServiceCollectionExtensions.cs [hunk 5]
```diff
@@ -0,0 +1,25 @@
+﻿using MatchedLearnerApi.Configuration;
+using MatchedLearnerApi.Interfaces;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace MatchedLearnerApi.Extensions
+{
+    public static class IServiceCollectionExtensions
```
[the remark this answers is not in the document]
Comment (ChughAkshay) [reply src/MatchedLearnerApi/Extensions/IServiceCollectionExtensions.cs:8]: ✔

File: src/MatchedLearnerApi/Extensions/IWebHostBuilderExtensions.cs [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MatchedLearnerApi.Configuration;
+using Microsoft.AspNetCore.Hosting;
+using SFA.DAS.Configuration.AzureTableStorage;
+
+namespace MatchedLearnerApi.Extensions
+{
+    public static class IWebHostBuilderExtensions
```
[the remark this answers is not in the document]
Comment (ChughAkshay) [reply src/MatchedLearnerApi/Extensions/IWebHostBuilderExtensions.cs:11]: ✔

File: src/MatchedLearnerApi/Program.cs (hunk 1 above)
Comment (ChughAkshay) [reply src/MatchedLearnerApi/Program.cs:9]: Not sure about this, it's worth double checking with Carl / Tom

**Comment by carlbm** [comment]
Superceded

**Pull request closed without merging**